package/kernel/mac80211/patches/983-rt2x00-add-r-calibration.patch

```diff
@@ -0,0 +1,193 @@
+--- a/drivers/net/wireless/ralink/rt2x00/rt2800lib.c
++++ b/drivers/net/wireless/ralink/rt2x00/rt2800lib.c
+@@ -7704,6 +7704,160 @@ void rt2800_rf_self_txdc_cal(struct rt2x
+ }
+ EXPORT_SYMBOL_GPL(rt2800_rf_self_txdc_cal);
+ 
++int rt2800_calcrcalibrationcode(struct rt2x00_dev *rt2x00dev, int d1, int d2)
++{
++	int calcode;
++	calcode = ((d2 - d1) * 1000) / 43;
++	if ((calcode%10) >= 5)
++		calcode += 10;
++	calcode = (calcode / 10);
++
++	return calcode;
++}
++EXPORT_SYMBOL_GPL(rt2800_calcrcalibrationcode);
++
++void rt2800_r_calibration(struct rt2x00_dev *rt2x00dev)
++{
++	u32 savemacsysctrl;
++	u8 saverfb0r1, saverfb0r34, saverfb0r35;
++	u8 saverfb5r4, saverfb5r17, saverfb5r18;
++	u8 saverfb5r19, saverfb5r20;
++	u8 savebbpr22, savebbpr47, savebbpr49;
++	u8 bytevalue = 0;
++	int rcalcode;
++	u8 r_cal_code = 0;
++	char d1 = 0, d2 = 0;
++	u8 rfvalue;
++	u32 MAC_RF_BYPASS0, MAC_RF_CONTROL0, MAC_PWR_PIN_CFG;
```

Comment:
kernel style prohibits all-capitable variable names. please change this and ideally use `scripts/check-patch.pl` to fix style issues prior to submission.